setup/ci/jenkins.md

```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
 after a bake and deploy stage from a pipeline instead of doing it manually.
 
-### Assumptions
-Here are a few assumptions we make in the following directions:
+### Prerequisites
 
-You have a running Spinnaker instance, with access to configuration files.
+In order to configure a Script stage, you need:
 
-You have a running Jenkins instance at `<jenkins_host>`, with a user profile set up with admin access.
+*   A running Spinnaker instance, with access to configuration files.
+*   A running Jenkins instance at `$JENKINS_HOST`, with a user profile set up
+    with admin access.
 
 ### Configure Jenkins
-`ssh` into your Jenkins machine.
 
-`wget` or `curl` the [raw job xml config file](https://storage.googleapis.com/jenkins-script-stage-config/scriptJobConfig.xml).
+1.  `ssh` into your Jenkins machine.
 
-To create the Jenkins job, run:
+2.  Download the [raw job xml config
+    file](https://storage.googleapis.com/jenkins-script-stage-config/scriptJobConfig.xml)
+    with the command:
 
-```
-curl -X POST -H "Content-Type: application/xml" -d @scriptJobConfig.xml \
-"http://<username>:<user_api_token>@<jenkins_host>/jenkins/createItem?name=<JOB_Name>"
-```
+    ```bash
+    curl -X GET \
+        -o "scriptJobConfig.xml" \
+        "https://storage.googleapis.com/jenkins-script-stage-config/scriptJobConfig.xml"
+    ```
+
+3.  Create the Jenkins job where your script will run. To do this, you need the
+    following information:
+
+    *   `$JENKINS_HOST`: your running Jenkins instance.
+    *   `$JOB_NAME`: the name of the Jenkins job where your script runs.
+    *   `$USER`: your Jenkins username.
+    *   `$USER_API_TOKEN`: the API token for your user. You can find this in
+        Jenkins in the **Configure** page for your user.
 
-where `<JOB_NAME>` is the name of the Jenkins job you create, e.g. "runSpinnakerScript"
-and `<user_api_token>` is the API token for your user, located at "/user/<username>/configure".
+    Then, run the command:
 
-In the job config in the Jenkins UI, set the GitHub repository containing your scripts as
-well as the git credentials.
+    ```bash
+    curl -s -XPOST 'http://$JENKINS_HOST/createItem?name=$JOB_NAME' \
+        -u $USER:$USER_API_TOKEN
+        --data-binary @scriptJobConfig.xml \
+        -H "Content-Type:text/xml"
+    ```
 
-In the UI, go to `"Manage Jenkins"` >> `"Configure System"` and set your git `user.name` and `user.email`.
+4.  Navigate to Jenkins >> the job you just created >> **Configure** and do two
+    things:
 
-At this point, you should be able to manually run the script job in Jenkins
-(with parameters) and see it succeed.
+    1.  Add the GitHub repository containing your scripts.
+    2.  Either create a `Spinnaker` node in which Jenkins will run all Script
+        jobs, or de-select the **Restrict where this project can be run**
+        checkbox.
 
-### Summary
+    At this point, you should be able to manually run the script job in Jenkins
```

Comment:
s/should be able to/can/ 
Or else, under what conditions might I not be able to?